apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/listeners/SasquatchAnalyticsListener.java

```diff
@@ -0,0 +1,75 @@
+package com.microsoft.appcenter.sasquatch.listeners;
```

Comment:
Fork of the file as jCenter dependency does not have CommonSchemaEventLog type yet.
Easier than using reflection.
Will merge the files after release.